addons/web/static/tests/fields/relational_fields_tests.js

```diff
@@ -2255,9 +2255,9 @@ QUnit.module('relational_fields', {
         form.destroy();
     });
 
-    QUnit.test('one2many list: unlink one record', function (assert) {
-        assert.expect(5);
-        this.data.partner.records[0].p = [2, 4];
+    QUnit.test('one2many list: unlink two record', function (assert) {
```

Comment:
I'm not sure I understand how any of the below tests actually work. How can they understand that they are in mobile view to trigger all the conditions that make the fullscreen modal appear while still retaining the normal remove behaviour in the very same test ?

Reply:
These changes are for regular FormViewDialog (on Desktop), to make sure that newly added 'Remove' button works as expected.

Reply:
So there is no test that forces the mobile view ? We should probably add a new test that forces config.isMobile to true in order to test the new feature.

Reply:
Right. I've added a test for mobile also.